Migration.md

```diff
@@ -0,0 +1,325 @@
+
+### Payload example
+#### v0:
+```
+{
+  "file": {
+    "name": "some_name"
+  }
+}
+```
+#### v1:
+```
+{
+  "seismic_store": {
+    "name": "some_name"
+  },
+  "include_headers": true
+}
+```
+
+Additionally, [`seismic`](https://docs.cognite.com/dev/guides/sdk/seismic/api/v1/#com.cognite.seismic.v1.Seismic) objects also have text headers. By default they are inherited from the original file, but may be overridden by the data managers. They can be fetched from [`com.cognite.seismic.v1.SeismicService.SearchSeismics`](https://docs.cognite.com/dev/guides/sdk/seismic/api/v1/#metadata-queries).
+
+### Payload example
+#### v1:
+```
+{
+  "seismic_store": {
+    "name": "some_name"
+  },
+  "include_text_header": true
+}
+```
+
+## com.cognite.seismic.Query.GetFileDataCoverage
+Use [`com.cognite.seismic.v1.SeismicService.SearchSeismicStores`](https://docs.cognite.com/dev/guides/sdk/seismic/api/v1/#metadata-queries) specifying [`coverage`](https://docs.cognite.com/dev/guides/sdk/seismic/api/v1/#searchseismicstoresrequest) with the [`format`](https://docs.cognite.com/dev/guides/sdk/seismic/api/v1/#com.cognite.seismic.v1.CoverageSpec) desired.
+
+### Payload example
+#### v0:
+```
+{
+  "file": {
+    "name": "some_name"
+  },
+  "in_wkt": true
+}
+```
+#### v1:
+```
+{
+  "file": {
+    "name": "some_name"
+  },
+  "coverage": {
+    "format": 0
+  }
+}
+```
+
+Additionally, [`seismic`](https://docs.cognite.com/dev/guides/sdk/seismic/api/v1/#com.cognite.seismic.v1.Seismic) objects also have coverage. They can be fetched from [`com.cognite.seismic.v1.SeismicService.SearchSeismics`](https://docs.cognite.com/dev/guides/sdk/seismic/api/v1/#metadata-queries).
+
+### Payload example
+#### v1:
+```
+{
+  "seismic_store": {
+    "name": "some_name"
+  },
+  "coverage": {
+    "format": 0
+  }
+}
+```
+
+## com.cognite.seismic.Query.GetFileLineRange
+Use [`com.cognite.seismic.v1.SeismicService.SearchSeismicStores`](https://docs.cognite.com/dev/guides/sdk/seismic/api/v1/#metadata-queries) with [`include_extent`](https://docs.cognite.com/dev/guides/sdk/seismic/api/v1/#searchseismicstoresrequest) set to `true`.
+
+### Payload example
+#### v0:
+```
+{
+  "file": {
+    "name": "some_name"
+  }
+}
+```
+#### v1:
+```
+{
+  "file": {
+    "name": "some_name"
+  },
+  "include_extent": true
+}
+```
+
+Additionally, [`seismic`](https://docs.cognite.com/dev/guides/sdk/seismic/api/v1/#com.cognite.seismic.v1.Seismic) objects also have the line range. They can be fetched from [`com.cognite.seismic.v1.SeismicService.SearchSeismics`](https://docs.cognite.com/dev/guides/sdk/seismic/api/v1/#metadata-queries).
+
+### Payload example
+#### v1:
+```
+{
+  "seismic": {
+    "id": "1234"
+  },
+  "include_line_range": true
+}
+```
+
+## com.cognite.seismic.Query.GetCrosslinesByInline
```

Comment:
This needs the full extent, though.

Reply:
Right. This one (and the one below) is very precise for that given line.
I redirected the user to `GetFileLineRange` because that's what the v0 was telling the user to do. But it seems like we can provide what the user wants with v1 by using `include_extent`.

I suppose I can bring the above solution down to these guys and rewrite the above one in terms of GetTraceBoudns.